recipe/meta.yaml

```diff
@@ -1,19 +1,20 @@
 {% set name = "cookiecutter" %}
-{% set version = "2.5.0" %}
+{% set version = "2.6.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e61e9034748e3f41b8bd2c11f00d030784b48711c4d5c42363c50989a65331ec
+  sha256: db21f8169ea4f4fdc2408d48ca44859349de2647fbe494a9d6c3edfc0542c21c
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv
   entry_points:
-    - cookiecutter = cookiecutter.cli:main
+    - cookiecutter = cookiecutter.__main__:main
+  skip: True  # [py<37]
```

Comment:
This is ok but not super necessary since we don't build anything less than 3.8 (which we'll also probably stop doing soon).

Reply:
I suppose it's good to include what upstream does though.

Reply:
Thanks! I was just about to say that was the only reason I included it. And I know it's not a strict rule, but the consensus among the team seems to be to match upstream.

Reply:
You are right @Jrice1317. I think It's preferable to stick to what upstream says.